### DISCUSSION TITLE
Unable to sign into Firefly after changing email

### DISCUSSION THREAD

**Discussion by Subliminalseducer** [firefly-iii/firefly-iii#8144] [Firefly III] [answered]
I'll start off by saying I'm relatively new to all of this,  so I'm sorry if I haven't provided enough information. I'm happy to provide what ever additional info is needed (I just might need help getting that info).

I currently have firefly 3 running as a docker container on a PC. Everything was working great until I decided to change the email I am using. The email I originally set up is no longer an existing email address. I was never able to get firefly to send an email, so I wasn't that concerned about it. I changed the email address to a new email, and firefly signed me out and asked to sign back in with my account information. When I try to sign in, I get an error "Until you verify your email address, you cannot login.", but a verification email was never sent because I couldn't get it working. 

It also appears the account is now blocked when I look at the logs:
`[2023-11-09 00:50:17] local.WARNING: User is blocked, cannot use authentication method.`

I tried signing in with my old email address, but get "Error! These credentials do not match our records."

I have tried looking around online but everything I could find was either a different issue altogether, or wasn't for the container version. Again, I apologize for being a bit new, but I appreciate any help. Thanks!

**Comment by JC5** [comment]
You need to confirm or cancel the email address change, and the links are in the email messages. 

If you haven't set that up your account is indeed blocked. 

I'll see what I can add to the documentation about this.

**Reply by Subliminalseducer** [reply]
Thanks!! Is there a way to work around this?

**Reply by JC5** [reply, marked as answer]
https://docs.firefly-iii.org/firefly-iii/faq/other/

**Reply by Subliminalseducer** [reply]
I ended up reinstalling it and starting from scratch (was kind of needed anyway), but this is very helpful. Thank you so much!
